lxd/instance/instancetype/instance.go

```diff
@@ -912,7 +912,7 @@ var InstanceConfigKeysVM = map[string]func(value string) error{
 	// Enabling this option prevents the use of some features that are incompatible with it.
 	// ---
 	//  type: bool
-	//  defaultdesc: `false`
+	//  defaultdesc: `true`
```

Comment:
But that isn't correct now, or am I misunderstanding? It's `true` when you are on a shared backend, but `false` otherwise (at least that's what the docs say :wink: ).

Reply:
Or actually, the docs don't say that it's `false`, but they don't say that the default has changed in that case.

Reply:
You're right, I'm removing this.

Reply:
We should specify this here - this IS documentation. ;) And that's why the field is called `defaultdesc` and not just `default`, because it allows text.

Reply:
fixed